schemas/common_definitions.json

```diff
@@ -10,6 +10,10 @@
     "description": "A question code used by downstream systems to identify answers.",
     "pattern": "^[0-9a-z]+$"
   },
+  "populated_string": {
+    "type": "string",
+    "pattern": "\\w+"
```

Comment:
Rather than restricting to word values, should be try and match a non-whitespace character i.e `\\S+`?

Reply:
This would allow us to use non-word chars in options - perhaps we would never do it, but it was possible to do that in the past.

Reply:
@samiwel We do need to allow non-word characters ie apostrophes. However, this only ensures that there is at least 1 word character in the string to be classed as valid. So a string with non-word character with atleast 1 word character would pass.

Reply:
Yes that's correct, Mebin. Labels containing apostrophes and other punctuation should be fine so long as they also contain at least one word character.

The more I think about it, I think this pattern is fine for now. Just because it's valid to have a label with a single character in it, doesn't mean we (the EQ team) would allow such a schema to be made live. It's more of a safety net to catch cases where the label may have accidentally not been specified.

Tom, ignore my previous comment regarding the pattern happy to give my approval to this change.